go/chat/sync.go

```diff
@@ -255,6 +255,15 @@ func (s *Syncer) sync(ctx context.Context, cli chat1.RemoteInterface, uid gregor
 			// Send notifications for a successful partial sync
 			s.SendChatStaleNotifications(ctx, uid, s.getConvIDs(incr.Convs), true)
 		}
+
+		// Queue background conversation loads
+		if s.G().ConvLoader != nil {
```

Comment:
I can actually just drop this altogether as `setupTest` creates a `ConvLoader` now.  (It didn't when I wrote this line)